app-arch/xz-utils/xz-utils-5.2.1-r1.ebuild

```diff
@@ -28,7 +28,7 @@ HOMEPAGE="http://tukaani.org/xz/"
 # See top-level COPYING file as it outlines the various pieces and their licenses.
 LICENSE="public-domain LGPL-2.1+ GPL-2+"
 SLOT="0"
-IUSE="nls static-libs +threads"
+IUSE="kernel_FreeBSD nls static-libs +threads"
```

Comment:
Wouldn't userland_* be more correct here? Assuming the `sha256.h` header is userland, not kernel stuff :).